composer.json

```diff
@@ -0,0 +1,47 @@
+{
+    "name": "cpms/common",
```

Comment:
Can you change the name to dvsa/mot-cpms-common? There might be more places where the old name is used, it would be good to make sure it's consistent :)